src/platform_impl/macos/window.rs

```diff
@@ -1210,3 +1213,11 @@ unsafe fn set_max_inner_size<V: NSWindow + Copy>(window: V, mut max_size: Logica
         window.setFrame_display_(current_rect, NO)
     }
 }
+
+// Converts from top-left coordinates to bottom-left coordinate system
+fn window_position(position: LogicalPosition<f64>) -> NSPoint {
+    NSPoint::new(
+        position.x,
+        CGDisplay::main().pixels_high() as f64 - position.y,
+    )
+}
```

Comment:
On top of the change in the wording of the documentation, I would suggest moving this function to `util/mod.rs` right underneath `bottom_left_to_top_left` (Which really doesn't have a good name or description in my opinion, but you don't need to be concerned about that.)